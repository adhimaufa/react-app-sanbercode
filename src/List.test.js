import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import TodoList from './Components/pageTodoList';

test(' list', () => {
  const component = renderer.create(
     <TodoList />
  );

  let tree = component.toJSON;
  expect(tree).toMatchSnapshot();

  // component.getInstance().handleNewTodo();
  // tree = component.toJSON();
  // expect(tree).toMatchSnapshot();
});
