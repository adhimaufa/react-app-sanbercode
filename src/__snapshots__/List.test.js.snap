// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` list 1`] = `[Function]`;
